docs/internal-developers/testing/releases/890.md

```diff
@@ -0,0 +1,124 @@
+3. Save.
+4. Visit the post/page via incognito mode and be sure that all the buttons have the same style.
+
+### Product button: Fix 'In cart' button localization ([7504](https://github.com/woocommerce/woocommerce-blocks/pull/7504))
+
+1. Add an instance of the All Products Block to the page.
+2. Use a plugin like Loco Translate to translate the text of %d in cart within the WooCommerce core plugin language files.
+3. Visit the All Products Block page and add a product to the cart.
+4. Verify the string is being translated.
+
+### Fix Mini Cart Block global styles ([7515](https://github.com/woocommerce/woocommerce-blocks/pull/7515))
+
+1. Enable TT3 theme.
+2. Go to Appearance > Site Editor and replace the header with `WooCommerce Essential Header | Light` pattern.
+3. Save and visit the frontend.
+4. Hover over the Mini Cart and be sure that the font size doesn't change.
+5. Edit some settings related to the Mini Cart Global Styles.
+6. Be sure that the settings are reflected correctly on the editor and frontend side.
+
+### Make the header and footer patterns text visible on all themes ([7524](https://github.com/woocommerce/woocommerce-blocks/pull/7524))
+
+1. Create a new page and insert all the WooCommerce Header and Footer patterns (dark & light, 12 patterns in total)
+2. Save, go to the frontend and check the text and icons are visible.
+3. Change the theme to TT3 and check again. Test with a few more different TT3 theme styles.
+
+### Active Filters: Fix active filter grouping for Ratings and Stock Status ([7577](https://github.com/woocommerce/woocommerce-blocks/pull/7577))
+
+1. Create a new page, add the all filters pattern and the All Products block.
+2. Add some products that contain ratings and various stock statuses.
+3. Filter by rating and stock status.
+4. Confirm that the active filters are grouped under one heading.
+
+### Experiment: Add Rate Limits to Store API ([5962](https://github.com/woocommerce/woocommerce-blocks/pull/5962))
+
+1. Enable Rate Limiting by using the options filter
+2. Try to apply a coupon or access `/wp-json/wc/store/cart` beyond current limits (currently 25 requests under 10 seconds)
+3. Ensure you get presented with an error "_Too many requests. Please wait xx seconds before trying again_."
+
+### Prevent padding from placeholder instructions to be overriden ([7552](https://github.com/woocommerce/woocommerce-blocks/pull/7552))
+
+1. Add a “Filter Products by Attribute” block
+2. Notice that the text "Display a list of filters based on a chosen attribute" is NOT too close to the separator
+
+## Experimental
+
+### [Product Query] Fix filter block data counter ([7257](https://github.com/woocommerce/woocommerce-blocks/pull/7257))
+
+1. Add a Product Query and enable ´Show only products on sale´.
+2. Add the Filter Products by Price block and add the Filter Products by stock block.
+3. Save and on the frontend be sure that the max price is $55.00 and there isn't the option "out of stock".
```

Comment:
Experimental features aren't available in the plugin release, should we link the nightly release here? Or should we remove this section? I lean toward to the latter as this is the testing note for 8.9.0.